src/main/resources/templates/progress/stage.html

```diff
@@ -2,10 +2,10 @@
 <html lang="cz" xmlns:th="http://www.thymeleaf.org">
 <head>
     <meta charset="UTF-8">
-    <title>Stanoviště ID - Postup</title>
+    <title>Šifra - Stav</title>
 </head>
 <body>
-<h1>Stanoviště číslo ID - Postup</h1>
+<h1>Šifra číslo <span th:text="${cypher.name}">cypherName</span> - Stav</h1>
```

Comment:
Neni tady to `cypherName` navic? Jinak bych dal asi jen `<h1>Šifra <span...` vzhledem k tomu, ze tam vypisujes jmeno a ne cislo.

Reply:
copy&paste z hint/list.html souboru - upraveno